code/modules/codex/entries/_codex_entry.dm

```diff
@@ -20,10 +20,15 @@
 	var/list/categories
 
 	///Allows you to mark a type as "abstract" and to not generate it.
-	var/abstract_type
+	abstract_type = /datum/codex_entry
 
 /datum/codex_entry/New(_display_name, list/_associated_paths, list/_associated_strings, _lore_text, _mechanics_text, _antag_text, _controls_text, _disambiguator)
 
+#ifdef UNIT_TESTS
+	if(isabstract(src))
+		CRASH("Attempted to instantiate abstract codex entry [type]")
+#endif
+
```

Comment:
codex entries are only instantiated in one place during init, this check is redundant